util/src/main/java/io/grpc/util/MultiChildLoadBalancer.java

```diff
@@ -357,12 +335,12 @@ public final ChildLbState getChildLbStateEag(EquivalentAddressGroup eag) {
   }
 
   /**
-   * Filters out non-ready and deactivated child load balancers (subchannels).
+   * Filters out non-ready child load balancers (subchannels).
    */
   protected final List<ChildLbState> getReadyChildren() {
     List<ChildLbState> activeChildren = new ArrayList<>();
     for (ChildLbState child : getChildLbStates()) {
-      if (!child.isDeactivated() && child.getCurrentState() == READY) {
```

Comment:
I don't see how deactivated children in ClusterManagerLoadBalancer get excluded from the ready children and are not actively used.

Reply:
`ClusterManagerLoadBalancer.acceptResolvedAddresses()` injects deactivated children back into the config, and deactivates them when it first notices they are removed. When it notices they are added back to the config, it reactivates them. That was really the only substantive change for ClusterManagerLb, as it moved that logic from MultiChildLb to ClusterManagerLb.

So both active and deactive children are passed down to MultiChildLB. `updateOverallBalancingState()` was already ignoring deactivated children for ConnectivityState calculations and the picker.